### ISSUE TITLE
Service to repeat last notification

### ISSUE THREAD

**Issue by macbih** [nimroddolev/chime_tts#129]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/nimroddolev/chime_tts/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

It happens to me occasionally that I miss a notification or have not fully understood it. I would like to be able to repeat the last notification in exactly the same way by using just a simple statement, for example in an automation behind a button.

### Describe the solution you'd like

It would be helpful if there was a service that allowed me to repeat the exact last `chime_tts.say` announcement including all parameters, e.g. something like `chime_tts.say_repeat`:

```
sequence:
  - service: chime_tts.say_repeat
    data: {}
```

An additional approach could be to let `chime_tts.say` generate an unique notification ID as `response_variable` which can be used for repeating the correct notification just in case there are many announcements:

```
sequence:
  - service: chime_tts.say_repeat
    data:
      notification_id: <GUID>
```

### Describe alternatives you've considered

It would be possible to (partially) achieve this by using `chime_tts.say_url` as well. But that would require not only to save the URL  but also all the other parameters including the target speaker(s) before.

**Comment by nimroddolev** [comment]
Hi @macbih. I like the idea and I hope to implement support it in an upcoming version.

Thanks for your suggestion 👍

**Comment by nimroddolev** [comment]
This feature has been included in the latest stable release, v1.1.0.
